### ISSUE TITLE
the pretrained model

### ISSUE THREAD

**Issue by xizi** [davidsandberg/facenet#330]
hi,@davidsandberg, what's the training style of the provide model?  Triplet loss training or Classifier training

**Comment by davidsandberg** [comment]
Hi @xizi,
The pretrained model has been trained using [softmax](https://github.com/davidsandberg/facenet/wiki/Classifier-training-of-inception-resnet-v1).

**Comment by xizi** [comment]
ok, thanks
